Platform/AlderlakeBoardPkg/CfgData/CfgData_Fusa_Feature.dlt

```diff
@@ -0,0 +1,77 @@
+## @file
+#
+#  FuSa feature Configuration Delta File.
+#
+#  Copyright (c) 2023, Intel Corporation. All rights reserved.<BR>
+#  SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+##
+#
+
+# Note: This file will come into effect only when ENABLE_FUSA = 1 in BoardConfig file
+
+# BIST Enables
+FUSA_CFG_DATA.FusaPeriodicScanBist0 | 1
+FUSA_CFG_DATA.FusaPeriodicScanBist1 | 1
+FUSA_CFG_DATA.FusaPeriodicArrayBist0 | 1
+FUSA_CFG_DATA.FusaPeriodicArrayBist1 | 1
+
+# Module Lockstep
+FUSA_CFG_DATA.Module0Lockstep | 0
+FUSA_CFG_DATA.Module1Lockstep | 0
+
+# Parity Options
+FUSA_CFG_DATA.DisplayFusaConfigEnable | 1
+FUSA_CFG_DATA.GraphicFusaConfigEnable | 1
+FUSA_CFG_DATA.OpioFusaConfigEnable | 1
+FUSA_CFG_DATA.IopFusaConfigEnable | 1
+FUSA_CFG_DATA.PsfFusaConfigEnable | 1
+FUSA_CFG_DATA.McParity | 1
+FUSA_CFG_DATA.IbeccParity | 1
+FUSA_CFG_DATA.CmfParity | 1
+
+# IBECC
+MEMORY_CFG_DATA.Ibecc | 1
+# IBECC operation mode: 0:Protect base on address range, 1:Non-protected, 2:All protected
+MEMORY_CFG_DATA.IbeccOperationMode | 2
+
+# CrashLog
+SILICON_CFG_DATA.CpuCrashLogEnable | 1
+SILICON_CFG_DATA.CrashLogReporting | 1
+
+# PROCHOT
+POWER_CFG_DATA.BiProcHot | 0
+POWER_CFG_DATA.DisableProcHotOut | 1
+POWER_CFG_DATA.ProcHotResponse | 1
+POWER_CFG_DATA.ProcHotLock | 1
+
+# TCC
+TCC_CFG_DATA.TccEnable   | 1
+
+# TCC Power profile settings
+# These are set automatically if TCC is enabled above.
+MEMORY_CFG_DATA.DisPgCloseIdleTimeout         | 1
```

Comment:
Do you want these settings to be applied even when Tcc is disabled? Or are these for info purposes only because Tcc is set to 1 by default?

Reply:
Got some clarification offline. Thanks.